docs/tutorials/heatmaps.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>Optional nits: "The heatmaps maintain" -&gt; "Heatmaps maintain", "constructor. e.g." -&gt; "constructor, e.g." (period to comma)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
done